src/gis/supported_formats.md

```diff
@@ -0,0 +1,218 @@
+# Supported Formats
+<!-- reference -->
+  
+[[toc]]
+
+## Vector data 
+
+
+| Format         | <MobileAppName /> (Android, iOS)  | QGIS (Windows, macOS, Linux)  | Note                                                                  |
+|----------------|-----------------------------------|-------------------------------|-----------------------------------------------------------------------|
+| GeoPackage     | :heart_eyes:                      | :heart_eyes:                  | Collaborative editing via <GitHubRepo id="lutraconsulting/geodiff" /> |
+| Shapefile      | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |
+| Delimited text | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          | 
+| Virtual layer  | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |                 
+| PostGIS        | :thumbsup:                        | :thumbsup:                    | Requires internet connection                                          |
+| WFS            | :thumbsup:                        | :thumbsup:                    | Requires internet connection                                          |
+
```

Comment:
I would add under the table that there are plenty of other formats supported by GDAL/OGR

for full list check `ogrinfo --formats`